Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -6,8 +8,9 @@ namespace
 }
 
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
-: initialStateOfChargePercent_(initialStateOfChargePercent)
-{
+: initialStateOfChargePercent_(initialStateOfChargePercent),
```

Comment:
Usually, multiple inline member initialization is formatted like
```
: foo_(foo)
, bar_(bar)
```

comma on the next line, so each line has something preceding it